packages/web/src/components/composer-input/ComposerInput.tsx

```diff
@@ -138,7 +142,7 @@ export const ComposerInput = (props: ComposerInputProps) => {
 
   const getTimestamps = useCallback(
     (value: string) => {
-      if (!track) return []
+      if (!track || !hasStreamAccess) return []
```

Comment:
On TrackScreen there is also a check for if the track is usdc gated. Idk if that's needed in this case too? https://github.com/AudiusProject/audius-protocol/blob/main/packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx#L241

Reply:
yea i saw that too, but figured that the stream access was the important part of the check there. I'll touch base with money team to make sure 👍

Reply:
Found out there there is a new meta and that tracks will always have an access prop that tells if the current user has stream access